.github/workflows/weekly_pull_requests.yaml

```diff
@@ -32,7 +50,7 @@ jobs:
                 uses: shivammathur/setup-php@v2
                 with:
                     php-version: 7.4
```

Comment:
@TomasVotruba should it be 7.3 same as code coverage action?

Reply:
I think 7.4 is ok here